plugins/frontend/backstage-plugin-github-insights/src/apis/GithubApi.ts

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
This `apis` directory is copied directly from the homepage markdown plugin. I will create a follow up path to refactor the markdown plugin to use components exported from here.